scripts/tests_and_coverage.sh

```diff
@@ -31,7 +31,8 @@ set +u
 
 if [ -z "${GITHUB_ACTIONS}" ] || [ "$1" == "test_and_coverage" ] ; then
     # Python tests (rerun with debug if failures)
-    PYTHONPATH=$(pwd) pytest -qx tests/ --cov='.' --no-cov-on-fail --cov-report term-missing || PYTHONPATH=$(pwd) pytest -xrpP --log-level debug tests/
```

Comment:
shouldn't we revert this or do you want debug by default?

Reply:
Yes, will be reverted once most of the work is done and detailed debugging is no longer needed.